locales/en.json

```diff
@@ -0,0 +1,283 @@
+        "links": {
+          "owml": "Read the OWML docs to learn how to make mods",
+          "template": "Outer Wilds mod template"
+        }
+      },
+      "newhorizons": {
+        "title": "Create custom content with New Horizons",
+        "description": {
+          "1": "With",
+          "2": ", you can create new planets and edit existing ones. Create new stories with character dialogue, translatable text, ship log entries, etc. You can make custom content without any coding knowledge, but you can also add your own code to further improve your creations."
+        },
+        "links": {
+          "docs": "Read the New Horizons docs"
+        }
+      },
+      "outerwilds": {
+        "title-questionmark": "?",
+        "description": "Outer Wilds is a neat game. Check it out and buy it or whatever.",
+        "links": {
+          "website": "Official website"
+        }
+      }
+    }
+  },
+  "error": {
+    "title": "Error - Outer Wilds Mods",
+    "description": "Something went wrong on the Outer Wilds Mods website"
+  },
+  "history": {
+    "timeline": {
+      "bigbang": "The Big Bang",
+      "spaceworthy": "Spaceworthy",
+      "alpha": "Outer Wilds Alpha",
+      "release": "Outer Wilds released",
+      "taimatem": "TAImatem joins",
+      "alek": "Alek joins",
+      "owml": "First OWML release",
+      "_nebula": "_nebula joins",
+      "nexusmods": "Nexus Mods page created",
+      "raicuparta": "Raicuparta joins",
+      "nomaivr": "First NomaiVR release",
+      "logan-email": "Logan emails",
+      "qsb": "First QSB release",
+      "modders-join": "Modders join OWML team",
+      "marshmallow": "Marshmallow",
+      "modmanager": "Mod manager",
+      "moddatabase": "Mod database",
+      "modswebsite": "Mods website",
+      "alpha-modding": "Alpha modding",
+      "eotw-leak": "DLC leak",
+      "eotw-announce": "DLC announced",
+      "johncorby": "JohnCorby",
+      "logan-discord": "Logan joins Discord",
+      "eotw-release": "DLC releases",
+      "xen": "Xen",
+      "newhorizons": "New Horizons",
+      "discord": "Modding Discord server",
+      "present": "Present",
+      "universe-death": "Heat death of the Universe",
+      "controls": {
+        "previous": "Previous",
+        "next": "Next",
+        "transitions-time": "Transitions: ",
+        "scale": "Scale: ",
+        "selected-count": "Selected: ",
+        "revealed-count": "Revealed: ",
+        "mods-count": "Mods: "
+      }
+    }
+  },
+  "months": {
+    "january": "January",
+    "february": "February",
+    "march": "March",
+    "april": "April",
+    "may": "May",
+    "june": "June",
+    "july": "July",
+    "august": "August",
+    "september": "September",
+    "october": "October",
+    "november": "November",
+    "december": "December"
+  },
+  "jam": {
+    "common": {
+      "results": "Results",
+      "podium": {
+        "first": "🥇 First place",
```

Comment:
I was trying out different things, but I believe Arabic might need reversed order for the emojis, which means it would not be part of the translation either way, because there is already a CSS property that manages that.
Changing when we get a good grasp of the whole situation.